packages/core/src/subgraph/queries/beneficiary.ts

```diff
@@ -0,0 +1,57 @@
+import { gql } from 'apollo-boost';
+
+import { BeneficiarySubgraph } from '../../interfaces/ubi/beneficiary';
+import { client } from '../config';
+
+export const getBeneficiaries = async (
+    communities: string[]
+): Promise<BeneficiarySubgraph[]> => {
```

Comment:
I'm afraid that this end up consuming all the RAM possible

Reply:
added batchs